docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -123,6 +123,7 @@ Basic layers:
 
 Basic layers:
 
+* _batch-jberet_: Tech preview support for Jakarta Batch. Optionally depends on _cdi_.
```

Comment:
Seems that you changed to be non optional

Reply:
Ah, yeah; thanks!

Reply:
Fixed